nixos/modules/services/misc/autosuspend.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.autosuspend;
+  settingsFormat = pkgs.formats.ini {};
+in {
+  options.services.autosuspend = {
+    enable = mkOption {
```

Comment:
Are you familiar with `mkEnableOption` at all?

Reply:
I wasn't. To be honest I just followed an example from Nix Pills on how to create a basic module, and mkEnableOption wasn't covered there

Reply:
It's just a convenience function, not required.